### PR TITLE
Mirror interface data to HTML elements

### PR DESCRIPTION
**Pull request by queengooborg**
This PR mirrors the data from the interfaces for HTML elements to the HTML elements themselves. This helps us reach 100% real values for the HTML category.

### CODE REVIEW

File: html/elements/col.json [hunk 1]
```diff
@@ -109,15 +108,14 @@
...
-                "version_added": null
+                "version_added": "1"
               },
               "chrome_android": "mirror",
               "edge": {
                 "version_added": "12"
               },
               "firefox": {
-                "version_added": false,
-                "notes": "See <a href='https://bugzil.la/2212'>bug 2212</a>."
```
Comment (Elchi3) [inline html/elements/col.json:120]: hm. this bug is still open?

File: html/elements/col.json [hunk 2]
```diff
@@ -40,15 +40,14 @@
...
-                "version_added": null
+                "version_added": "1"
               },
               "chrome_android": "mirror",
               "edge": {
                 "version_added": "12"
               },
               "firefox": {
-                "version_added": false,
-                "notes": "See <a href='https://bugzil.la/915'>bug 915</a>."
```
Comment (Elchi3) [inline html/elements/col.json:51]: the bug was wontfixed? So, I guess it still doesn't work?

**Review by Elchi3 — COMMENTED** [review]
Looks good but I am skeptical about all the changes for firefox from `false` to a version number when there is an unresolved bug in the notes.

File: html/elements/col.json (hunk 2 above)
Comment (queengooborg) [reply html/elements/col.json:51]: It's possible, but it'd be interesting for the property to be defined on the interface if it doesn't...

Comment (queengooborg) [reply html/elements/col.json]: I went ahead and reverted these changes so they can be researched at a later time!

**Review by Elchi3 — APPROVED** [review]
Great thank you, the rest seems reasonable to me. 👍

**Pull request merged**